src/win32/modbus.vcproj

```diff
@@ -233,46 +232,55 @@
 			/>
 			<Tool
 				Name="VCMIDLTool"
-				TargetEnvironment="3"
 			/>
 			<Tool
 				Name="VCCLCompilerTool"
 				Optimization="0"
+				EnableIntrinsicFunctions="true"
 				WholeProgramOptimization="false"
-				AdditionalIncludeDirectories="$(SolutionDir)"
-				PreprocessorDefinitions=""
+				AdditionalIncludeDirectories="..\src;..;.;D:/include/msvc_std"
```

Comment:
That seems weird to me to use hard path names...

Reply:
No idea how that slipped in there! In the `.vcproj` I agree, I'll see to fix this